CustomPlugin/CpStatusController.cs

```diff
@@ -35,17 +35,17 @@ public CpStatusBody CheckUp()
             return CheckUpHandler();
         }
 
-        public virtual CpStatusBody HealthinessHandler()
+        protected virtual CpStatusBody HealthinessHandler()
```

Comment:
For simplicity, can't we just `return CpStatus.Ok();` in the `Healthz` function?

Reply:
We can't because we couldn't override the base method anymore since it's a controller route.

Reply:
that's right, my bad.